tests/conftest.py

```diff
@@ -23,6 +23,8 @@ def pytest_collection_modifyitems(items):
         "test_sql_storage_mapping",
         "test_postgres",
         "test_snowflake",
+        "test_postgres_annotation",
+        "test_duckdb_annotation",
```

Comment:
@theferrit32 I can't seem to locate the `test_duckdb_annotation.py` script. I think it's missing. I also believe there should be a `test_duckdb.py` script for the non-annotation test cases for the duckdb database.

Reply:
@larrybabb included in new commits